dockerfile

```
@@ -1,25 +1,31 @@
-    
-    # Copy go.mod and go.sum first and download dependencies
-    COPY src/go.mod src/go.sum ./
-    RUN go mod download
-    
-    # Copy the remaining source code
-    COPY src/ .
-    
-    # Build the Go app (static binary if possible)
-    RUN CGO_ENABLED=0 GOOS=linux go build -o main ./cmd/api
+# Install build dependencies for CGO and SQLite
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)